frontend/app/chat/[chatId]/components/Dialog.tsx

```diff
@@ -1,15 +1,16 @@
 import { useChatContext } from "@/lib/context";
 
-import { ChatMessages } from "./ChatMessages";
+import { MessagesDialog } from "./MessagesDialog";
 import { ShortCuts } from "./ShortCuts";
 
 export const ChatDialog = (): JSX.Element => {
-  const { history } = useChatContext();
+  const { messages, notifications } = useChatContext();
 
-  const shouldDisplayShortcuts = history.length === 0;
+  const shouldDisplayShortcuts =
+    messages.length === 0 && notifications.length === 0;
```

Comment:
notifications of Done status only here. We still want to show the shortcuts if we only have a "Pending ...." notification in the chat page